### PR TITLE
GitHub

### PR DESCRIPTION
**Pull request by AgrimAsthana-CS**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,13 @@
   "name": "@c12e/cortex-studio-skill-lab-extension",
   "version": "0.4.4",
   "description": "A JupyterLab extension by CognitiveScale",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CognitiveScale/cortex-cli.git"
```
Comment (dschueller-cs) [inline package.json]: Is this the correct url?

Comment (AgrimAsthana-CS) [reply package.json]: Will correct it..

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+- lts/*
+cache:
+  directories:
+  - node_modules
+deploy:
+  provider: npm
+  email: jgielstra@cognitivescale.com
```
Comment (dschueller-cs) [inline .travis.yml]: Shouldnt this be some kind of company email address, rather than to Johan?

Comment (AgrimAsthana-CS) [reply .travis.yml]: Currently only Johan and Matt have access to push to npm..

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  skip_cleanup: true
+  on:
+    branch: master
+  api_key:
+    secure: TODO
```
Comment (daldridge-cs) [inline .travis.yml]: Doesn't this need to be specified (TODO -> ?)?

Comment (AgrimAsthana-CS) [reply .travis.yml]: when we merge develop --> master we need the token to be present.
since the master will push to NPM and develop will push to Jfrog.

Comment (daldridge-cs) [reply .travis.yml]: Right... but can we obtain the token and put it in this branch?  Or must it be done in the develop or master branch?  Johan indicated it was a couple of CLI commands...
```
travis login --github-token <GIT HUB TOKEN> --debug
travis encrypt <NPM TOKEN> --add deploy.api_key
````
Looks like maybe only Johan or Matt can run those commands, but I think we should have it setup by the time it's merged into develop.

**Pull request merged**